site/src/components/WorkspacesTable/WorkspacesTableBody.tsx

```diff
@@ -0,0 +1,69 @@
+import Button from "@material-ui/core/Button"
+import Link from "@material-ui/core/Link"
+import TableCell from "@material-ui/core/TableCell"
+import TableRow from "@material-ui/core/TableRow"
+import AddCircleOutline from "@material-ui/icons/AddCircleOutline"
+import { FC } from "react"
+import { Link as RouterLink } from "react-router-dom"
+import { workspaceFilterQuery } from "../../util/filters"
+import { WorkspaceItemMachineRef } from "../../xServices/workspaces/workspacesXService"
+import { EmptyState } from "../EmptyState/EmptyState"
+import { TableLoader } from "../TableLoader/TableLoader"
+import { WorkspacesRow } from "./WorkspacesRow"
+
+export const Language = {
+  emptyCreateWorkspaceMessage: "Create your first workspace",
+  emptyCreateWorkspaceDescription: "Start editing your source code and building your software",
```

Comment:
```suggestion
  emptyCreateWorkspaceDescription: "Start editing your source code and building your software.",
```

Since we use punctuation in the other descriptions.